wasm.md

```diff
@@ -718,15 +738,16 @@ The allocation of a new `tableinst`. Currently at most one table may be defined
        andBool MAX <=Int #maxTableSize()
 
     rule <k> table { _ _ } => trap ... </k>
-         <tabAddrs> MAP </tabAddrs> requires MAP =/=K .Map
+         <tabIndices> MAP </tabIndices> requires MAP =/=K .Map
 
     rule <k> table { MIN MAX } => . ... </k>
-         <tabAddrs>    .Map => (0 |-> NEXT)  </tabAddrs>
-         <nextTabAddr> NEXT => NEXT +Int 1 </nextTabAddr>
+         <nextTabIdx> NEXTIDX => NEXTIDX +Int 1 </nextTabIdx>
+         <tabIndices> .Map => (NEXTIDX |-> NEXTADDR) </tabIndices>
+         <nextTabAddr> NEXTADDR => NEXTADDR +Int 1 </nextTabAddr>
          <tabs>
            ( .Bag
           => <tabInst>
-               <tAddr>   NEXT </tAddr>
+               <tAddr>   NEXTADDR </tAddr>
```

Comment:
Formatting, line up the closing cell tags.

Reply:
fixed